# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchTypes.ipdlh
# Commit: c0cb3c17f246
# Full Hash: c0cb3c17f2461a19203f3b295a6c56b417483b2a
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 84162d09eef8 -r c0cb3c17f246 dom/fetch/FetchTypes.ipdlh
--- a/dom/fetch/FetchTypes.ipdlh	Thu Jan 12 15:09:45 2023 +0000
+++ b/dom/fetch/FetchTypes.ipdlh	Thu Jan 12 15:09:45 2023 +0000
@@ -35,8 +35,9 @@
   nsID uuid;
 };
 
-struct ParentToChildStream {
-  RemoteLazyInputStream stream;
+union ParentToChildStream {
+  RemoteLazyInputStream;
+  IPCStream;
 };
 
 struct ChildToParentStream {
@@ -86,6 +87,8 @@
   nsCString alternativeDataType;
   nsITransportSecurityInfo securityInfo;
   PrincipalInfo? principalInfo;
+  nsCString bodyBlobURISpec;
+  nsString bodyLocalPath;
 };
 
 struct ParentToParentInternalResponse {